### ISSUE TITLE
[Bug]: exposing functions for formulas

### ISSUE THREAD

**Issue by theMostCreativeUserName** [RafaelGB/obsidian-db-folder#856]
### What happened?

the exported function has to be names the same as the file name, to be allegible to be used in the database.
- multiple functions cannot be exported from the same file
- a function can only be exported if its name is the same as the filename (even if it is the only function present)

### What platform were you using?

Desktop

### Version of the plugin

3.3.2

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by donovanrost** [comment]
Are you able actually use a function in a formula?

**Comment by theMostCreativeUserName** [comment]
> Are you able actually use a function in a formula?

Yes, I can use a function, but only if the fuction has the same name as the file I saved it as.
Even if I export a function with a different name explicitly, I cant use it in a formula

**Comment by RafaelGB** [comment]
hello @theMostCreativeUserName , we can open a discussion here. The way the formulas are included is like this, you need to have the name of the file
